doc/design/assumeutxo.md

```diff
@@ -77,7 +77,9 @@ original chainstate remains in use as active.
 Once the snapshot chainstate is loaded and validated, it is promoted to active
 chainstate and a sync to tip begins. A new chainstate directory is created in the
 datadir for the snapshot chainstate called
-`chainstate_[SHA256 blockhash of snapshot base block]`.
+`chainstate_[SHA256 blockhash of snapshot base block]`. When this directory is present
```

Comment:
isn't this outdated now after d59db4ff4b585d2b86b68ea0db206ec1c0cc5022?

Reply:
Oops, good catch.